TotalMayhemGames/EscapeRoom
# Feat/pressure plate puzzle

How to test:
- Add the SpikePressurePlatePuzzle.prefab to the scene 
- Walk with a player over the starting slab 
- Spikes should reset the player to the slab
- The capsule should change color once started
